auto_gptq/utils/marlin_utils.py

```diff
@@ -93,33 +95,33 @@ def _get_cached_marlin_save_name(model_name_or_path):
     If local:    {model_name_or_path}/autogptq_model.safetensors
     """
     if os.path.isdir(model_name_or_path):
-        return os.path.join(model_name_or_path, "autogptq_model.safetensors")
+        return os.path.join(model_name_or_path, MARLIN_CACHE_FILE)
     else:
         namespace, subfolder = model_name_or_path.split("/")
         assets_path = huggingface_hub.cached_assets_path(
             library_name="auto_gptq", namespace=namespace, subfolder=subfolder
         )
-        return os.path.join(assets_path, "autogptq_model.safetensors")
+        return os.path.join(assets_path, MARLIN_CACHE_FILE)
```

Comment:
There is actually a hidden bug here. if user manually update the weights, the cache file is not auto-cleared. This would require a md5 type check of the weights so cache can be invalidated but doing md5 on a huge checkpoint still reads the entire checkpoint.